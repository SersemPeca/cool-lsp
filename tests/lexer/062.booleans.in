True
TRue
TrUe
TruE
TRUe
TrUE
TRuE
TRUE
