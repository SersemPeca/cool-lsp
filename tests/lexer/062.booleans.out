#1 TYPEID True
#2 TYPEID TRue
#3 TYPEID TrUe
#4 TYPEID TruE
#5 TYPEID TRUe
#6 TYPEID TrUE
#7 TYPEID TRuE
#8 TYPEID TRUE
